.github/workflows/ci.yml

```diff
@@ -36,6 +36,7 @@ jobs:
         DEBIAN_FRONTEND: noninteractive
       # [ACTION REQUIRED] Add your build dependencies here
       run: |
+        sudo apt-get update
         sudo apt-get install -y cppcheck afl
```

Comment:
Perhaps my diagnosis was just wrong: https://packages.ubuntu.com/search?keywords=afl suggests that we'd  have to use `afl++` on Ubuntu 20.04 (it's only 20.10 that includes the transitional package `afl`, which would have made for a smooth transition from 18.04, which did have `afl`).